src/features/revurdering/revurderingoppsummering/formuevilkåroppsummering/FormuevilkårOppsummering.tsx

```diff
@@ -65,14 +65,14 @@ export const Formuevurdering = ({ vurdering }: { vurdering: VurderingsperiodeFor
                                 {intl.formatMessage({ id: 'gjeldendeformue.eps' })}
                             </Normaltekst>
                             <div className={classNames(styles.formueVerdier, styles.formueInfo)}>
-                                <Element>{vurdering.grunnlag?.epsFormue.verdiIkkePrimærbolig}</Element>
-                                <Element>{vurdering.grunnlag?.epsFormue.verdiEiendommer}</Element>
-                                <Element>{vurdering.grunnlag?.epsFormue.verdiKjøretøy}</Element>
-                                <Element>{vurdering.grunnlag?.epsFormue.innskudd}</Element>
-                                <Element>{vurdering.grunnlag?.epsFormue.verdipapir}</Element>
-                                <Element>{vurdering.grunnlag?.epsFormue.pengerSkyldt}</Element>
-                                <Element>{vurdering.grunnlag?.epsFormue.kontanter}</Element>
-                                <Element>{vurdering.grunnlag?.epsFormue.depositumskonto}</Element>
+                                <Element>{vurdering.grunnlag.epsFormue.verdiIkkePrimærbolig}</Element>
```

Comment:
Vi kan fjerne nullable fra typen. Når det kommer til akkurat formuevilkår så er vurdering og grunnlag 1 til 1. Dersom det finnes en vurdering, finnes det og et grunnlag.

Reply:
Dersom vurderingene er en tom liste betyr det at den ikke er vurdert, noe som ikke skal skje ved revurdering.
> vurderinger = when (this) {
            is Vilkår.Formue.IkkeVurdert -> emptyList()
            is Vilkår.Formue.Vurdert -> vurderingsperioder.map { it.toJson() }
        }